pnpm-lock.yaml

```diff
@@ -13740,10 +13756,10 @@ snapshots:
     dependencies:
       '@aws-crypto/sha256-browser': 5.2.0
       '@aws-crypto/sha256-js': 5.2.0
-      '@aws-sdk/client-sso-oidc': 3.609.0(@aws-sdk/client-sts@3.637.0)
+      '@aws-sdk/client-sso-oidc': 3.609.0(@aws-sdk/client-sts@3.609.0)
       '@aws-sdk/client-sts': 3.609.0
       '@aws-sdk/core': 3.609.0
-      '@aws-sdk/credential-provider-node': 3.609.0(@aws-sdk/client-sso-oidc@3.609.0(@aws-sdk/client-sts@3.609.0))(@aws-sdk/client-sts@3.609.0)
+      '@aws-sdk/credential-provider-node': 3.609.0(@aws-sdk/client-sso-oidc@3.609.0(@aws-sdk/client-sts@3.609.0))(@aws-sdk/client-sts@3.637.0)
```

Comment:
@mojotalantikite I had a bad merge and reverted. But when I merge main in and install, it changes this part of the lock file. I'm not sure why this differs from main as I did'n't change anything with these libraries. Is this going to be an issue?

Reply:
hmm, I'm not sure why in this specific case, but I'd guess that maybe there are some transitive dependencies that might specify `client-sts` somewhere? I'd have to dig in to check, but honestly I don't think it'll be a problem. As long as CI is passing I'd say we're fine!